### ISSUE TITLE
Chinese cotton links to RoW low voltage electricity supplier

### ISSUE THREAD

**Issue by mfastudillo** [OcelotProject/Ocelot#119]
Hello hello

I have noticed something a bit strange. using the default system model, some datasets seem not to link to the right market. I tested it for Chinese cotton, and for some reason it uses RoW low voltage electricity instead of the Chinese one. The allocated datasets provided by ecoinvent do link to the Chinese electricity market.

I am using the unlinked version of ecoinvent given during the ocelot workshop, that could be the reason of the strange linking behaviour. Attached a cumbersome notebook to test this.  

Miguel

[ocelot_test_linking.pdf](https://github.com/OcelotProject/Ocelot/files/504475/ocelot_test_linking.pdf)

**Comment by cmutel** [comment]
TLDR: This needs #100 to be fixed.

Ecoinvent provides markets for the [China Southern power grid](https://en.wikipedia.org/wiki/China_Southern_Power_Grid) and the [State Grid Corporation of China](https://en.wikipedia.org/wiki/State_Grid_Corporation_of_China), and a market group for all of China. Because we haven't got around to implementing market groups yet, the cotton process needs to consume from a market which completely covers the location `CN`, which neither grid does - therefore it defaults to `RoW`. When #100 is closed, the market group for low voltage electricity in China will be available, and will be linked correctly, as it [is in the ecoinvent release](https://v32.ecoquery.ecoinvent.org/Details/UPR/3ec170de-d216-403f-ba67-5bc541f19300/290c1f85-4cc4-4fa1-b0c8-2cb7f4276dce).

**Comment by cmutel** [comment]
Will close this when #100 is also closed.

**Comment by cmutel** [comment]
Fixed in 04a71515ff428b598e223fadceb829d6314f89ed. However, more work needs to be done before this branch is merged. See commit message.
